### PR TITLE
Release 0.37.5

### PR DESCRIPTION
**Pull request by karlb**
I didn't realize that the manifest was *that* selective.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -4,11 +4,12 @@ include raiden_contracts/constants.py
 include raiden_contracts/data*/contracts.json
 include raiden_contracts/data*/deployment_kovan.json
 include raiden_contracts/data*/deployment_goerli.json
+include raiden_contracts/data*/deployment_goerli_unstable.json
 include raiden_contracts/data*/deployment_rinkeby.json
 include raiden_contracts/data*/deployment_ropsten.json
 include raiden_contracts/data*/deployment_mainnet.json
 include raiden_contracts/data*/deployment_services_kovan.json
-include raiden_contracts/data*/deployment_services_goerli.json
```
Comment (palango) [inline MANIFEST.in:11]: This shoudn't be deleted, right?

Comment (karlb) [reply MANIFEST.in:11]: Definitively not. Fixed. Not my best day, apparently.

Comment (palango) [reply MANIFEST.in:11]: Hehe, it's weekend soon!

**Review by palango — APPROVED** [review]
Looks good now.

**Pull request merged**